c1: 0.1,
c2: 0.1,
c3: 0.1,
c4: 0.1,
t1: 0.1,
t2: 0.1,
s1: 0.1,
s2: 0.1,
k0: 0.1,
kg: 0.1,
hover_throttle: 0.1,
max_force: 0.1,
mass: 0.1,
k_speed: 0.1,
search_range: 0.1
